### ISSUE TITLE
Clarify radii and distance types in documentation

### ISSUE THREAD

**Issue by t4im** [minetest/minetest#2235]
Please provide clarification in the documentation whether distances in
```lua
minetest.get_objects_inside_radius(pos, radius)
minetest.find_node_near(pos, radius, nodenames)
{ max_hear_distance=… }
-- this is most likely a maximum metric:
minetest.find_path(pos1,pos2,searchdistance,max_jump,max_drop,algorithm) 
```
are based on manhattan, maximum or euclidean metrics.
It's not unreasonable to assume that objects might use another metric than nodes.

**Comment by t4im** [comment]
After a short code exploration with @Zeno- we found 

`minetest.get_objects_inside_radius(pos, radius)` and `{ max_hear_distance=… }` to use an euclidian metric

`minetest.find_path(pos1,pos2,searchdistance,max_jump,max_drop,algorithm)` to use a maximum metric

and `minetest.find_node_near(pos, radius, nodenames)` to use a manhattan metric

jackpot! ;-)

**Comment by t4im** [comment]
Addressed by #3192
